types/index.ts

```diff
@@ -19,57 +19,119 @@ import { expectTypeOf } from 'expect-type'
 Basic usage examples
 */
 {
-  const math = create(all);
+  const math = create(all)
 
   // functions and constants
-  math.round(math.e, 3);
-  math.round(100.123, 3);
-  math.atan2(3, -3) / math.pi;
-  math.log(10000, 10);
-  math.sqrt(-4);
-  math.pow([[-1, 2], [3, 1]], 2);
-  const angle = 0.2;
-  math.add(math.pow(math.sin(angle), 2), math.pow(math.cos(angle), 2));
+  math.round(math.e, 3)
+  math.round(100.123, 3)
+  math.atan2(3, -3) / math.pi
+  math.log(10000, 10)
+  math.sqrt(-4)
+  math.pow(
+    [
+      [-1, 2],
+      [3, 1],
+    ],
+    2
+  )
+  const angle = 0.2
```

Comment:
That last expression seems a bit egregious to me. Is there any way to rein in the line breaking at all?

Reply:
I think the most readable way to write this that Prettier will tolerate is
```ts
const A = [
    [-1, 2],
    [3, 1],
  ]
  math.pow(A, 2)
```

@josdejong @gwhitney How annoying do you think the inability to write matrices on a single line is? Definitely something to think about more if you ever seriously consider using prettier on the src/ codebase. [Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEB6AVAHSgBQE5wwwCWceABACZwC2AhlJQM7kwAWdM5NETXUcPnErk6ePHQCeLAOYRy0bgFcANiQAOKuORXEBTAHTYcEeLGJ0VKyaKhQIkvTPIAzCBSb0r3TnmJhBcj1yAFlONgApAGVsdFRsSCg+cgBBcgBecgBtLIBGABoAJgBdfOyAZnyAFmLi7GxUVHJ1AiJSPABaYhl7AgToZIAhDOy8otKK6tr6qEbyey52YhYASXIZJUEmfqSuAGERnIBWWpB8kAh1EgHkUDE8CAB3HDEEJmQQS0epd-OAIwkYAA1oQonQaHAADJ6ODIFyWJhwf6AkEwKLqOhgJzIGB4TbnWh-YTUSiQhgbOgyOAAMXc9DaUBkHzoShgEDOIDYMBoKgA6mxiPAmBiAlE3oLiAA3QWSD5gJi-EB6RF4GD4Sn0OEIpEgABWTAAHlEnFoAIpKUywpDwlSI84YvAqj5-OhElQclp6GC84iUdjIAAcAAZ7Q9EbyJOoPi1BGRJbDzgBHC3wfCXd4oOhMDoCYTCDkEZPEAjqmSa63a86ImjEHF4nVME1wc2WrW2nUwV0+v1sZCFc64ujEXSMvYQGjlkCCI4cpSIgAqrozNrtIElmxWjAQaLAfiuKUYURgki0bcRAF9z0A)

My intuition is that 
- usually matrices look better on multiple lines anyway
- but this could be annoying if testing, e.g., `math.pow(matrix, n)` with lots of different small matrices...

Reply:
Yeah, we're a bit limited in not being able to put a small nested array on a single line, but it's not _that_ bad, I can live with it. In these particular tests ,we could extract the array `[[1,2,3], [4,5,6]]` that we reuse a lot and only define it once.